src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,9 @@
+@import "../variables.scss";
+
+.logo {
```

Comment:
let logo be an element of header, you have margins for it, and you can't add margins for blocks